SPECS/glib-networking/glib-networking.spec

```diff
@@ -1,55 +1,42 @@
 Summary:        Glib networking modules
 Name:           glib-networking
-Version:        2.59.1
-Release:        8%{?dist}
+Version:        2.70.0
+Release:        1%{?dist}
 License:        GPLv2+ WITH exceptions
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Development
 URL:            https://gitlab.gnome.org/GNOME/glib-networking/
-Source0:        http://ftp.gnome.org/pub/GNOME/sources/glib-networking/2.59/%{name}-%{version}.tar.xz
-Patch0:         CVE-2020-13645.patch
-BuildRequires:	autogen-libopts-devel
-BuildRequires:	openssl-devel
-BuildRequires:  glib
-BuildRequires:  glib-devel
-BuildRequires:  glib-schemas
-BuildRequires:  gnome-common
-BuildRequires:  gnutls-devel
-BuildRequires:  intltool
-BuildRequires:	libtasn1-devel
+Source0:        https://download.gnome.org/sources/%{name}/2.70/%{name}-%{version}.tar.xz
+BuildRequires:  ca-certificates
+BuildRequires:  gcc
+BuildRequires:  gettext
 BuildRequires:  meson
-BuildRequires:	nettle-devel >= 3.7.2
-BuildRequires:  ninja-build
-Requires:	gnutls
-Requires:	libtasn1
-Requires:	nettle >= 3.7.2
-Requires:	openssl
+BuildRequires:  glib2-devel
+BuildRequires:  gnutls-devel
+BuildRequires:  gnutls
+BuildRequires:  gsettings-desktop-schemas-devel
+BuildRequires:  p11-kit-devel
+Requires:       ca-certificates
```

Comment:
Why are we adding `ca-certificates` to `BuildRequires` and `Requires`? It shouldn't be needed during build time (potentially test build might need it, but the we should enclose it with `%if %{with_check}`) and we should avoid have user install that package by accident.

Reply:
good catch, ca-certificates is not used.